vantage/manifest.json

```
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f9d211f9-8cd2-4bab-9718-462c5a3fa9ad",
```

```suggestion
  "app_uuid": "2784a986-2673-4189-a3b8-320755f6c2b4",
```
This `app_uuid` has to match the `published_app_uuid` field of your private version of the app in the developer platform.